index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello world!</h1>
```

Comment:
Add a basic structure to the html document

Reply:
done